tests/cases/italic/inline.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../../jats.rng" schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="../../../src/support.sch" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+
+<italic>
+    Text
+    <bold>Bold</bold>
+    <italic>Italic</italic>
```

Comment:
As it stands we'll end up with nested HTML `<i>`, which won't do anything visually. Probably ok not to do anything about this. /cc @davidcmoulton